django-backend/fecfiler/committee_accounts/views.py

```diff
@@ -198,7 +205,7 @@ def register_committee(committee_id, user):
         committee_id=committee_id
     ).first()
     if existing_account or f1_email != email:
-        raise BadRequest("could not register committee")
+        raise ValidationError("could not register committee")
```

Comment:
Would context on why the committee couldn't be registered be useful here? Because it could be there is an `existing_account` or `f1_email != email`

Reply:
The exception message was split up so that a more specific error message is now presented. The email check was moved up so to short circuit the need for a committee check should the email not match.